vmss/vmss-autoscale.yml

```diff
@@ -0,0 +1,46 @@
+# Description
+# ===========
+# Use this playbook to auto scale up VMSS.
+# Use vmss-create playbook to create VMSS first
+# This sample requires Ansible 2.7
+
+---
+- hosts: localhost
+  vars:
+    resource_group: "{{ resource_group_name }}"
+    vmss_name: vmsstest
```

Comment:
How about use "myVMSS"?

Reply:
let's extract a global variable file maybe in next pr. @zikalino